docs/components/input-controller/gpio.md

```diff
@@ -141,11 +128,11 @@ Click **Create component** and then fill in the attributes for your model:
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `board` | string | **Required**| The name of the board component with GPIO or ADC pins to use as the controlling device. |
+| `buttons` | object | **Required** | The [Buttons](../#button-controls) available for control. These should be connected to the GPIO/ADC board. <pre> **Each buttons's fields:** <br><br> <code>name</code>: Name of the Digital Interrupt the button/switch is connected to, as configured on the board component. <br><br> <code>control</code>: The [Control](../#control-field) type to use when reporting events as this button's state is changed. <br><br> <code>invert</code>: Boolean indicating if the digital input (high/low) should be inverted when reporting button Control value indicating button state. <br> This option is given because digital switches vary between high, `1`, and low, `0`, as their default at rest. <pre> *true:* `0` is pressed, and `1` is released. <br> *false (default):* `0` is released, and `1` is pressed. </pre> <code>debounce_ms</code>: How many milliseconds to wait for the interrupt to settle. This is needed because some switches can be electrically noisy. </pre> |
```

Comment:
```suggestion
| `buttons` | object | **Required** | The [buttons](../#button-controls) available for control. These should be connected to the GPIO/ADC board. <pre> **Each buttons's fields:** <br><br> <code>name</code>: Name of the Digital Interrupt the button/switch is connected to, as configured on the board component. <br><br> <code>control</code>: The [Control](../#control-field) type to use when reporting events as this button's state is changed. <br><br> <code>invert</code>: Boolean indicating if the digital input (high/low) should be inverted when reporting button Control value indicating button state. <br> This option is given because digital switches vary between high, `1`, and low, `0`, as their default at rest. <pre> *true:* `0` is pressed, and `1` is released. <br> *false (default):* `0` is released, and `1` is pressed. </pre> <code>debounce_ms</code>: How many milliseconds to wait for the interrupt to settle. This is needed because some switches can be electrically noisy. </pre> |
```

Reply:
Caps bc object name here!  (see Control). I think it was code cased at one point but it cluttered up this table even more